client/src/components/PostBody/PostBody.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./PostBody.css";
+
+// Function based React Component
+const PostBody = props => {
+  // Default Class to apply to Component
+  let classList = `PostBody`;
```

Comment:
If you're not updating or changing the class name, then you can just reference the class as a string in the JSX and remove this extra variable